### DISCUSSION TITLE
A few suggestions

### DISCUSSION THREAD

**Discussion by Rustly** [Pryaxis/TShock#1952] [Ideas]
Replace "my query" TSPlayer.SendMultipleMatchError with a more understandable message for people less experienced with TShock. For example: Quote names if they have spaces. eg ''Terra Blade''.

Put the tsCharacter table in a separate sqlite file.

Add config options to change the MySQL schema name for each database (eg add UsersDatabaseName to config) to allow users who do not have active devs to ease the use of Dimensions.

I had a few other things to note but I can't remember ATM, my bad.

**Comment by moisterrific** [comment]
I agree, it actually took me awhile to realize I need to encapsulate some parameters with “”. And the error message hints def could be clearer about the functions of `< >`  `[ ]` and `*` in commands. 

Really like the idea of splitting SSC database into a separate file, makes it a lot easier for server owners to reset everyone’s inventory when it’s time for a fresh world. Arguably you can do that with a plugin, but it’s nicer to have it out of the box.

**Comment by hakusaro** [comment]
I am going to opt to moving this into a discussion. Since it encapsulates multiple things, we really need to make separate issues for each one. This is a bit of a grab bag issue right now -- we can't close this with any one PR and it's hard to understand at a glance on the issue list.
